assets/i18n/fr.json

```diff
@@ -2,13 +2,40 @@
     "error": {
         "mqtt": "Désolé, une erreur s'est produite lors de la communication avec la plateforme Snips",
         "config": "Désolé, une erreur s'est produite avec votre fichier de configuration.",
-        "locale": "Désolé, une erreur s'est produite avec vos paramètres de localisation.",
+        "localisation": "Désolé, une erreur s'est produite avec vos paramètres de langue.",
```

Comment:
The key should be `locale` and not `localisation`, even for the french localisation file